### ISSUE TITLE
Incompatible with Sodium

### ISSUE THREAD

**Issue by Apollounknowndev** [CelestiumMC/confogurable#1]
https://discord.com/channels/1181103999336849428/1185312372626301020/1185318429557800980

**Comment by sleepingdragoninn** [comment]
pretty sure this has been closed a while

**Comment by Apollounknowndev** [comment]
yeah this is a long dead project just haven't gotten around to archiving the repo

**Comment by sleepingdragoninn** [comment]
Aw that's so sad! respack driven conditions for fog really does seem like the best overall solution - was it overshadowed by another project I should know about?

**Comment by Apollounknowndev** [comment]
I would recommend taking a look at [Fog](https://modrinth.com/mod/fog).

**Comment by sleepingdragoninn** [comment]
Hmm, ARR and most of their base ideas just look silly - I'll just stick with simple fog control, thanks though!
